src/main/java/gift/domain/Category.java

```diff
@@ -0,0 +1,73 @@
+package gift.domain;
+
+import gift.dto.CategoryDTO;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "category")
+public class Category {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "name", unique = true, nullable = false)
+    private String name;
+
+    @Column(name = "color", nullable = false)
+    private String color;
+
+    @Column(name = "imageUrl", nullable = false)
+    private String imageUrl;
+
+    @Column(name = "description", nullable = false)
+    private String description;
+
+    protected Category() {
+
+    }
+
+    public Category(String name, String color, String imageUrl, String description) {
+        this.name = name;
+        this.color = color;
+        this.imageUrl = imageUrl;
+        this.description = description;
+    }
+
+    public Category(Long id, String name, String color, String imageUrl, String description) {
+        this.id = id;
+        this.name = name;
+        this.color = color;
+        this.imageUrl = imageUrl;
+        this.description = description;
+    }
```

Comment:
아이디 유무에 따른 생성자를 다르게 한 부분 👍🏼
minor) 이부분 중복되는 부분은 this 호출해서 사용해볼 수 있지 않을까요?
```suggestion
    public Category(String name, String color, String imageUrl, String description) {
        this.name = name;
        this.color = color;
        this.imageUrl = imageUrl;
        this.description = description;
    }

    public Category(Long id, String name, String color, String imageUrl, String description) {
        this.id = id;
        this(name, color, imageUrl, description);
    }
```

Reply:
몇몇 엔티티엔 this() 생성자를 사용하지 않았네요..! 수정하였습니다